otter/models/interface.py

```diff
@@ -263,12 +267,14 @@ class IScalingGroup(Interface):
     uuid = Attribute("UUID of the scaling group - immutable.")
     tenant_id = Attribute("Rackspace Tenant ID of the owner of this group.")
 
-    def view_manifest(with_webhooks=False):
+    def view_manifest(with_policies=True, with_webhooks=False):
         """
         The manifest contains everything required to configure this scaling:
         the config, the launch config, and all the scaling policies.
 
-        :param bool with_webhooks: Should webhooks information be included?
+        :param bool with_policies: Should policies information be included?
+        :param bool with_webhooks: If policies are included, should webhooks
+            information be included?
```

Comment:
Non-blocking: Maybe it should raise `ValueError` if webhooks is asked without policies?